Platform-Linux/1-Ansible/ansible-jumpbox-connection.md

```diff
@@ -0,0 +1,40 @@
+There are multiple methods. The method I will be describing will be a method using the inventory file.
+All commands will be in the inventory file on the ansible controller
+
+# 1. Define a proxy/jumpbox host
+```ini
+[proxy]
+<ip of proxy> ansible_ssh_user=<username> ansible_ssh_private_key_file=<path/to/privkey>
+
+# 2. Add Machines you wish to connect to through the proxy. 
+# I was only able to get this to wor with pubkey authentication setup.
+[machines]
+<ip of machine 1> ansible_ssh_user=<username> ansible_ssh_private_key_file=<path/to/privkey>
+<ip of machine 2> ansible_ssh_user=<username> ansible_ssh_private_key_file=<path/to/privkey>
+<ip of machine 3> ansible_ssh_user=<username> ansible_ssh_private_key_file=<path/to/privkey>
+
+[machine:vars]
+ansible_ssh_common_args='-o ProxyCommand="ssh -i ~/.ssh/ccdc_harvester -p 22 -W %h:%p blueteam@192.168.1.49"'
```

Comment:
Maybe include a comment for what that is doing and where the files in it should be located

Reply:
What is a ansiblesshcommon arg, and how would you change the arguments there as the infrastructure changes. Where is the key file located?

Reply:
You made most of the commands generic except this one. Maybe include a file where all commands are generic and one concrete example?